content/authoring/guidelines/kata.mdx

```diff
@@ -3,33 +3,39 @@ title: Kata Authoring Guidelines
 kind: reference
 ---
 
-This article contains a set of guidelines, which can be used by authors to create good, enjoyable, high-quality kata. They were collected to help ensure that kata are of sufficient quality and users' experience will be as good as possible.
+import A from "@components/Anchored";
 
+This article contains a set of guidelines, which can be used by authors to create good, enjoyable, high-quality kata. They were collected to help ensure that kata are of sufficient quality and users' experience will be as good as possible.
 
 ## Idea
-  
-- **Make sure your kata adds some new value to Codewars.** Avoid creating blatant duplicates, and use the idea which has not yet been widely used to create a kata. Note that with so many existing kata, this can be very challenging to achieve, because it's very difficult to find out if your idea has already been used or not. Remember that you can always ask more experienced users if your idea is worthy of creating a new kata, or maybe it has been already exploited sufficiently.
-- **Do not use problems from other sites,** unless it's allowed by that site's licensing policy and/or terms of use. Explicitly give appropriate credit if your kata is based on, or inspired by, someone else's work.
- 
+
+- **<A id="novelty">Make sure your kata adds some new value to Codewars.</A>** Avoid
```

Comment:
Do I see correctly that long lines got broken down here? Is there any particular reason for that? Is it required to make anchors work?

Reply:
No, it was autoformatted by Prettier. Not required, but breaking a long line is nice because we'll get better diffs.